seekret/apitest/context/context.py

```diff
@@ -0,0 +1,127 @@
+import contextlib
+from typing import Any, Optional
+
+import requests
+from requests.structures import CaseInsensitiveDict
+
+from seekret.apitest.auth import create_auth
+from seekret.apitest.context.response import ResponseWrapper
+from seekret.apitest.runprofile import RunProfile
+
+
+def resolve_path_params(path: str, path_params: dict[str, Any]):
```

Comment:
this should fail if some parameters were not specified/ too many parameters were specified

Reply:
Fixed